CONTRIBUTING.md

```diff
@@ -25,6 +25,9 @@ gulp
 
 When you're ready, continue with next steps:
 
+- Run ```sh
+gulp uglify
```

Comment:
Actually, you need to run just `gulp`, or `gulp webserver test lint` to run tests and assure that the code has no lint errors.